src/components/tables/ChannelTable.vue

```diff
@@ -142,6 +159,28 @@ async function getData() {
     elements.value.push(...dataVersions as any)
     // console.log('count', count)
     total.value = count || 0
+
+    // Look for misconfigured channels
+    // This will trigger if the channel disables updates based on metadata + if the metadata is undefined
+    let anyMisconfigured = false
+    const channels = dataVersions
+      .filter(e => e.disableAutoUpdate === 'version_number')
+      .map(e => e as any as typeof element)
+
+    for (const channel of channels) {
+      if (channel.version.minUpdateVersion === null) {
+        channel.misconfigured = true
+        anyMisconfigured = true
+      }
+
+      if ((channel.enable_progressive_deploy || channel.enableAbTesting) && channel.secondVersion.minUpdateVersion === null) {
+        channel.misconfigured = true
+        anyMisconfigured = true
+      }
+    }
```

Comment:
you could use the `.map` instead of looping through the array twice

something like:
```suggestion
    .map(e => {
      const channel = e as any as typeof element

      if (channel.version.minUpdateVersion === null) {
        channel.misconfigured = true
        anyMisconfigured = true
      }
      if ((channel.enable_progressive_deploy || channel.enableAbTesting) && channel.secondVersion.minUpdateVersion === null) {
        channel.misconfigured = true
        anyMisconfigured = true
      }
      
      return channel
    })
```

Reply:
when we loop it only once, one thing that we could improve are the conditionals.

likely, we could use a `switch (true)` state to improve readability.

an example: 
```typescript
      switch (true) {
        case channel.version.minUpdateVersion === null:
        case channel.enable_progressive_deploy || channel.enableAbTesting:
        case channel.secondVersion.minUpdateVersion === null:
          channel.misconfigured = true
          anyMisconfigured = true
          break;
        default:
          break;
      }
```